tools/toolchains/arm.py

```diff
@@ -18,6 +18,7 @@
 from copy import copy
 from os.path import join, dirname, splitext, basename, exists
 from os import makedirs, write
+from shutil import copyfile
```

Comment:
Could you remove the unneeded import?

Reply:
Yeah sorry missed that one, just pushed an update